doc/api/readers/index.rst

```diff
@@ -23,6 +23,7 @@ Reader Classes
    :toctree: _autosummary
    :template: custom-class-template.rst
 
+    
```

Comment:
Minor comment: A newline was added here.  Can we instead get rid of the whitespace on the adjacent empty line?

Reply:
GitHub is weird about showing whitespace diff, so I'm not sure if I've got this right.

Reply:
Line 26?

Reply:
Line 25 looks to have spaces to me in some views on GitHub.  If we remove those, do we need the added newline in this PR?

Reply:
No not really. Btw from what i can see. Line 25 is empty. Full blank